spec/ngsild_entity_schema.json

```diff
@@ -0,0 +1,120 @@
+{
+    "definitions": {
+        "EntityParams": {
+            "type": "object",
+            "properties": {
+                "location": {
+                    "$ref": "ngsild_schema.json#/definitions/CProperty"
+                },
+                "observationSpace": {
+                    "$ref": "ngsild_schema.json#/definitions/CProperty"
+                },
+                "operationSpace": {
+                    "$ref": "ngsild_schema.json#/definitions/CProperty"
+                },
+                "@context": {
+                    "type": ["string","object","array"]
+                }
+            }
+        },
+        "EntityMandatoryParams": {
+            "type": "object",
+            "required": ["type"],
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "minLength": 1
+                }
+            }
+        },
+        "Entity": {
+            "allOf": [
+                {
+                    "type": "object",
+                    "required": ["id", "dateCreated", "dateModified"],
```

Comment:
oh, we renamed those to `createdAt`, `modifiedAt`

Reply:
be careful, they are not required at creation time

Reply:
EntityDescription is intended to be used on creation time (they don't appear here since is information provided by the backend). Entity is intended to be the description used when receiving the information from the backend (EntityList o directly ent Entity if retrieved just one)